server/src/main/java/au/org/aodn/ogcapi/server/core/model/StacCollectionModel.java

```diff
@@ -7,6 +7,9 @@
 import java.util.List;
 import java.util.UUID;
 
+/**
+ * This is used to map the json from Elastic search to object
+ */
 @Data
 @Builder
 public class StacCollectionModel {
```

Comment:
hmm, so any changes to the `StacCollectionModel` to the indexer will require a sync with the same class in OGC API.

Reply:
this is just my comment